protocol/x/clob/keeper/deleveraging_test.go

```diff
@@ -1095,6 +1136,215 @@ func TestProcessDeleveraging(t *testing.T) {
 				*tc.offsettingSubaccount.GetId(),
 				uint32(0),
 				tc.deltaQuantums,
+				bankruptcyPriceQuoteQuantums,
```

Comment:
yeah I needed to push this into the if statement though, because the `GetBankruptcyPriceInQuoteQuantums` returns an error for my test cases where tc.expectedErr != nil. So the test code was erroring out before executing the ProcessDeleveraging logic. 

I am intentionally skipping this calculation for error cases.